README.md

```diff
@@ -18,8 +14,6 @@ ______________________________________________________________________
   <a href="#license">License</a>
 </p>
 
-<!-- DO NOT ADD CONDA DOWNLOADS... README CHANGES MUST BE APPROVED BY EDEN OR WILL -->
-
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pytorch-lightning)](https://pypi.org/project/pytorch-lightning/)
```

Comment:
all these links currently point to PyTorch Lightning
which ones should we keep, which ones to remove/replace?